src/Libraries/OrchardCore.Commerce.Abstractions/Models/Payment.cs

```diff
@@ -2,7 +2,7 @@
 using OrchardCore.Commerce.MoneyDataType;
 using System;
 
-namespace OrchardCore.Commerce.Payment.Models;
+namespace OrchardCore.Commerce.Abstractions.Models;
```

Comment:
`Abstractions` in the name space may be weird. `Payments` may be better

Reply:
This file was moved into the `OrchardCore.Commerce.Abstractions` library which contains all the shared models and interfaces needed across several modules, to avoid unnecessary cross dependency. We don't group namespaces by role inside the Abstractions library at all, because most classes here are hard to categorize and only doing it for some that are easy (like this one) would be even more weird. Also it feels like unnecessary namespace pollution to me.